daemon/daemon_unix.go

```diff
@@ -276,7 +276,7 @@ func verifyContainerResources(resources *containertypes.Resources) ([]string, er
 	}
 	if resources.OomKillDisable != nil && !sysInfo.OomKillDisable {
 		resources.OomKillDisable = nil
-		return warnings, fmt.Errorf("Your kernel does not support oom kill disable.")
+		logrus.Warnf("Your kernel does not support OomKillDisable, OomKillDisable discarded.")
```

Comment:
Should this also be appended to `warnings` (as we do with the other options below?)